thetis/utility.py

```diff
@@ -683,7 +683,8 @@ def anisotropic_cell_size(mesh):
     op2.par_loop(kernel, P0_ten.node_set, evectors.dat(op2.RW), evalues.dat(op2.RW), B.dat(op2.READ))
 
     # Return minimum eigenvalue
-    return interpolate(evalues[-1], FunctionSpace(mesh, "DG", 0))
+    dim = mesh.topological_dimension()
```

Comment:
Just curious: why did you hit this? Also the kernel above seems to be hard-coded to assume evalues has dim 2, so if the evalues vector function provided isn't dim 2 then I don't think that's safe. Would it not be simpler in any case to have the kernel just return the largest eigenvalue, since that's the only thing we're actually using?

Reply:
I think it was a UFL update (unrelated to this PR). But, yes, it would better to update as you suggest.